.teamcity/_self/lib/customBuildType/E2EBuildType.kt

```diff
@@ -145,8 +145,17 @@ open class E2EBuildType(
 					cd test/e2e
 					mkdir temp
 
+					# Disable exit on error to support retries.
+					set +o errexit
```

Comment:
@noahtallen @dpasque **comment:blocking** - while I understand there's nothing more permanent than a temporary solution, I am not comfortable adding a build step (and the associated overhead, however minor) to all Pre-Release Tests and Calypso E2E builds. 

What do you think? Would you both prefer to have a separate step for clear separation?

Reply:
My main question is just whether there is all that much overhead in an extra step -- it doesn't seem to take that much time in my experience, unless there is a different issue here.

That said, I don't think it matters that much. It's only really useful to have an extra step for people who debug the test suites a lot (e.g. us), so if you don't think it makes a big difference, I don't think we need to care about it too much

Reply:
I'm fine either way too!

> it doesn't seem to take that much time in my experience, unless there is a different issue here.

This was my experience too, but again, I don't think it really matters whether we make a separate build step or toggle the flags like you have it 😄